charts/tigera-operator/values.yaml

```diff
@@ -32,6 +32,9 @@ certs:
 # Resource requests and limits for the tigera/operator pod.
 resources: {}
 
+# Common labels for the all resources created by this chart
```

Comment:
sorry, the main typo here was "the all" - it should just be "all" (without "the")